public/docs/ts/latest/tutorial/toh-pt5.jade

```diff
@@ -412,6 +412,9 @@ code-example(format='').
   making private variables for both:
 +makeExample('toh-5/ts/app/hero-detail.component.ts', 'ctor', 'app/hero-detail.component.ts (constructor)')(format=".")
 :marked
+  We implement the `OnInit` interface.
```

Comment:
The `makeExample` extract that you added doesn't "implement the interface".  The implementation is the `ngOnInit`.

What your extract shows is  how we tell TypeScript that the class implements the interface.

Either we say it that way or leave this out. Thanks.